### ISSUE TITLE
`~` ($HOME) in path not supported by PyStemmusScope

### ISSUE THREAD

**Issue by BSchilperoort** [EcoExtreML/STEMMUS_SCOPE_Processing#34]
I tried to use the `~` character in the file paths when setting up the PyStemmusScope configuration, however, this does not seem to be supported. Instead I have to use the (full) absolute path.

Would we want to support this? `pathlib`'s `.expanduser()` will retrieve the home directory from `~`.

```py
>>> pth = pathlib.Path('~/STEMMUS_SCOPE_data/model_parameters/vegetation_property/')
>>> pth.expanduser()
PosixPath('/home/bart/STEMMUS_SCOPE_data/model_parameters/vegetation_property')
```

**Comment by SarahAlidoost** [comment]
yes, we can refactor [read_config](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/3f0ec439a9fdb34eb636480c9f176009a46db45e/PyStemmusScope/config_io.py#L15) to use [utils.to_absolute_path](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/3f0ec439a9fdb34eb636480c9f176009a46db45e/PyStemmusScope/utils.py#L33).
